hack/update-codecgen.sh

```diff
@@ -144,7 +144,7 @@ for current in "${index[@]}"; do
   # We use '-d 1234' flag to have a deterministic output every time.
   # The constant was just randomly chosen.
   ###echo "DBG: running ${CODECGEN} -d 1234 -o ${base_generated_file} ${base_file}"
-  ${CODECGEN} -d 1234 -o "${base_generated_file}" "${base_file}"
+  ${CODECGEN} -d 1234 -o "${base_generated_file}" -nr 'ObjectMeta' "${base_file}"
```

Comment:
why isn't the dependency sorting in update-codecgen.sh sufficent for this already? It starts by removing the generated files, so ObjectMeta shouldn't have a generated encoding implementation during generation.